src/test/ui/mismatched_types/cast-rfc0401.stderr

```diff
@@ -20,57 +20,67 @@ error: no field `f` on type `fn() {main}`
 75 |     let _ = main.f as *const u32;
    |                  ^
 
-error: non-scalar cast: `*const u8` as `&u8`
+error[E0605]: non-scalar cast: `*const u8` as `&u8`
```

Comment:
Any way to remove "non-scalar" from this? Your expanded error descriptions you added above use the term "primitive types", can that be used everywhere for all these errors instead of "non-scalar"? I never see "non-scalar" in any docs anywhere.